docs/userguide/execution.rst

```diff
@@ -113,7 +113,8 @@ Parsl provides a simple user-managed model for controlling elasticity.
 It allows users to prescribe the minimum
 and maximum number of blocks to be used on a given executor as well as
 a parameter (*p*) to control the level of parallelism. Parallelism
-is expressed as the ratio of task execution capacity and active tasks (running tasks + dependency resolved but waiting tasks)
+is expressed as the ratio of task execution capacity and active tasks (running tasks)
```

Comment:
Sorry to be nitpicky, but now that we've broken it up, why not call running tasks running tasks instead of active tasks?